src/renderer/scss/component/_shapeshift.scss

```diff
@@ -0,0 +1,48 @@
+// Can't think of a better way to do this
+// The initial shapeshift form is 311px tall
+// the .shapeshift__initial-wrapper class is only added when the form is being loaded
+// Once the form is rendered, there is a very smooth transition because the height doesn't change
+.shapeshift__wrapper.shapeshift__initial-wrapper {
+  min-height: 346px;
+}
+
+.shapeshift__content {
+  .spinner {
+    margin-top: $spacing-vertical * 3;
+  }
+}
+
+.shapeshift__tx-info {
+  min-height: 55px;
```

Comment:
If this is to avoid jumping, I'd rather just put placeholder text like "Calculating rates..."

Reply:
We would still need some sort of min height on that unless the heights are the same. I would rather not since this check is usually very quick and you would only see that for a small time before it goes back to the info text.